src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx

```diff
@@ -46,20 +51,20 @@ class TableBodyRow extends Component {
   };
 
   render() {
-    const { classes, rowData, columns, renderTd, expandLabel, trProps } = this.props;
+    const { classes, rowData, columns, renderTd, expandLabel, trProps, borderLessRows } = this.props;
```

Comment:
borderless is one word, so should be `borderlessRows`

Reply:
fixed